src/components/v2/V2Project/index.tsx

```diff
@@ -34,7 +34,22 @@ export default function V2Project({
     handle,
   } = useContext(V2ProjectContext)
 
-  const [activePage, setActivePage] = useState<ProjectPage>(selectedPage)
+  const [activePage, setActivePage] = useState<ProjectPage>(defaultPage)
+
+  const router = useRouter()
+  useEffect(() => {
+    setActivePage(() => {
+      switch (router.query.page) {
+        case 'info':
+          return 'info'
```

Comment:
**question** why is this preferrable to:
- /@handle/settings => settings page
- /@handle => project dashboard

I think this would be a more conventional approach.